plugins/outputs/nebius_cloud_monitoring/README.md

```diff
@@ -34,3 +34,55 @@ Folder ID from instance metadata. In this plugin we use [Google Cloud notation]
 This internal metadata endpoint is only accessible for VMs from the cloud.
 
 [Google Cloud notation]: https://nebius.com/il/docs/compute/operations/vm-info/get-info#gce-metadata
+
+### Reserved Labels
+
+Nebius Monitoring backend using json format to receive the metrics:
+
+```json
+{
+"name": "metric_name",
+"labels": {
+"key": "value",
+"foo": "bar"
+},
+"ts": "2023-06-06T11:10:50Z",
+"value": 0
+}
+```
+
+But key of label cannot be `name` because it's reserved for `metric_name`.
+
+So this payload:
+
+```json
+{
+"name": "systemd_units_load_code",
+"labels": {
+"active": "active",
+"host": "vm",
+"load": "loaded",
+"name": "accounts-daemon.service",
+"sub": "running"
+},
+"ts": "2023-06-06T11:10:50Z",
+"value": 0
+}
+```
+
+will be replaced with:
+
+```json
+{
+"name": "systemd_units_load_code",
+"labels": {
+"active": "active",
+"host": "vm",
+"load": "loaded",
+"_name": "accounts-daemon.service",
+"sub": "running"
+},
+"ts": "2023-06-06T11:10:50Z",
+"value": 0
+}
```

Comment:
Can we maybe prettify the JSON sections with some indentation to make them more readable?!

Reply:
done